src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/View/Controller/ExposureSubmissionQRInfoViewModel.swift

```diff
@@ -7,45 +7,154 @@ import UIKit
-						.icon(
-							UIImage(imageLiteralResourceName: "Icons_QR1"),
-							text: .string(AppStrings.ExposureSubmissionQRInfo.instruction1)
-						),
-						.icon(
-							UIImage(imageLiteralResourceName: "Icons_QR2"),
-							text: .attributedString(
-								AppStrings.ExposureSubmissionQRInfo.instruction2
-									.inserting(emphasizedString: AppStrings.ExposureSubmissionQRInfo.instruction2HighlightedPhrase)
-							)
-						),
-						.icon(
-							UIImage(imageLiteralResourceName: "Icons_QR3"),
-							text: .attributedString(
-								AppStrings.ExposureSubmissionQRInfo.instruction3
-									.inserting(emphasizedString: AppStrings.ExposureSubmissionQRInfo.instruction3HighlightedPhrase)
-							)
-						),
-						.icon(
-							UIImage(imageLiteralResourceName: "Icons_QR4"),
-							text: .string(AppStrings.ExposureSubmissionQRInfo.instruction3) // will be refactored!
-						)
-					]
+		var model = DynamicTableViewModel([])
+
+		// Ihr Einverständnis
+		model.add(
+			.section(
+			   header: .image(
+				   UIImage(imageLiteralResourceName: "Illu_Appinfo_Datenschutz_2"),
+				   accessibilityLabel: AppStrings.ExposureSubmissionQRInfo.imageDescription,
+				   accessibilityIdentifier: AccessibilityIdentifiers.ExposureSubmissionWarnOthers.accImageDescription,
+				   height: 250
+			   ),
+			   cells: [
+				.body(text: AppStrings.ExposureSubmissionQRInfo.titleDescription)
+			])
+		)
+
+		// Testergebnis abrufen
+		model.add(
+			.section(cells: [
+				.title2(text: AppStrings.ExposureSubmissionQRInfo.headerSection1,
+						accessibilityIdentifier: AccessibilityIdentifiers.ExposureSubmissionQRInfo.headerSection1),
+				.body(text: AppStrings.ExposureSubmissionQRInfo.bodySection1),
+				.icon(
+					UIImage(imageLiteralResourceName: "Icons_QR1"),
+					text: .string(AppStrings.ExposureSubmissionQRInfo.instruction1)
+				),
+				.icon(
+					UIImage(imageLiteralResourceName: "Icons_QR2"),
+					text: .attributedString(
+						AppStrings.ExposureSubmissionQRInfo.instruction2
+							.inserting(emphasizedString: AppStrings.ExposureSubmissionQRInfo.instruction2HighlightedPhrase)
+					)
 				)
-			)
+			])
+		)
+
+		// Helfen Sie mit, …
+		model.add(
+			.section(cells: [
+				.title2(text: AppStrings.ExposureSubmissionQRInfo.headerSection2,
+						accessibilityIdentifier: AccessibilityIdentifiers.ExposureSubmissionQRInfo.headerSection2),
+				.body(text: AppStrings.ExposureSubmissionQRInfo.bodySection2)
+			])
+		)
+
+		// 'Flags'
+		model.add(
+			.section(separators: .all, cells: [
+				.countries(countries: countries, accessibilityIdentifier: AccessibilityIdentifiers.ExposureSubmissionQRInfo.countryList)
+			])
+		)
+
+		// Ihr Einverständnis
+		model.add(
+			.section(cells: [
+				.acknowledgement(title: NSAttributedString(string: AppStrings.ExposureSubmissionQRInfo.acknowledgementTitle),
+								 description: NSAttributedString(string: AppStrings.ExposureSubmissionQRInfo.acknowledgementBody),
+								 bulletPoints: bulletPoints,
```

Comment:
The texts from ExposureSubmissionQRInfo_acknowledgement_1_1  to ExposureSubmissionQRInfo_acknowledgement_6 appear twice on the screen.

Reply:
Thanks! The cell was 'configured' twice and the stack view now properly reset. That's fixed now.